systems/plants/RigidBodyManipulator.h

```diff
@@ -212,6 +212,8 @@ class DLLEXPORT_RBM RigidBodyManipulator
   DrakeCollision::ElementId addCollisionElement(const RigidBody::CollisionElement& element, const std::shared_ptr<RigidBody>& body, std::string group_name);
 
   void updateCollisionElements(const std::shared_ptr<RigidBody>& body);
+  void getTerrainPoints(int body_id, Eigen::MatrixXd &terrain_points);
```

Comment:
we're trying to move away from using body_id and towards using RigidBody references/pointers. Do we really need this method?